kolibri/plugins/coach/assets/src/views/assignments/RecipientSelector.vue

```diff
@@ -24,14 +24,12 @@
 <script>
 
   import isEqual from 'lodash/isEqual';
-  import KButton from 'kolibri.coreVue.components.KButton';
   import KCheckbox from 'kolibri.coreVue.components.KCheckbox';
   import KRadioButton from 'kolibri.coreVue.components.KRadioButton';
 
   export default {
     name: 'RecipientSelector',
     components: {
-      KButton,
```

Comment:
was the linter able to identify that this component was unused?

Reply:
yeah, it's this rule vue/no-unused-components.